gqcpy/src/QCMethod/HF/GHF/GHFSCFEnvironment_bindings.cpp

```diff
@@ -0,0 +1,105 @@
+// This file is part of GQCG-GQCP.
+//
+// Copyright (C) 2017-2020  the GQCG developers
+//
+// GQCG-GQCP is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// GQCG-GQCP is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with GQCG-GQCP.  If not, see <http://www.gnu.org/licenses/>.
+
+#include "QCMethod/HF/GHF/GHFSCFEnvironment.hpp"
+
+#include <pybind11/eigen.h>
+#include <pybind11/pybind11.h>
+
+
+namespace py = pybind11;
+
+
+namespace gqcpy {
+
+
+void bindGHFSCFEnvironment(py::module& module) {
+    py::class_<GQCP::GHFSCFEnvironment<double>>(module, "GHFSCFEnvironment", "An algorithm environment that can be used with standard GHF SCF solvers.")
```

Comment:
For GHF, I think we're going to have to add the Python bindings for the `complex` type as well.

Could you open an issue in order to add this feature?